internal/venice-common/src/main/java/com/linkedin/venice/pubsub/manager/TopicManager.java

```
@@ -84,7 +84,8 @@ public TopicManager(String pubSubClusterAddress, TopicManagerContext context) {
         TopicManager.class.getSimpleName() + " [" + Utils.getSanitizedStringForLogger(pubSubClusterAddress) + "]");
     this.pubSubClusterAddress = Objects.requireNonNull(pubSubClusterAddress, "pubSubClusterAddress cannot be null");
     this.topicManagerContext = context;
-    this.stats = new TopicManagerStats(context.getMetricsRepository(), pubSubClusterAddress);
+    this.stats =
+        new TopicManagerStats(context.getMetricsRepository(), pubSubClusterAddress, context.getVeniceComponent());
```

should we fail if component is unknown ?